components/sections/Footer/index.tsx

```diff
@@ -0,0 +1,58 @@
+import classNames from 'classnames';
+import Image from 'next/image';
+import type { FC } from 'react';
+import { FormattedMessage } from 'react-intl';
+
+import NavItem from '@/components/sections/NavItem';
+import { useSiteConfig } from '@/hooks/useSiteConfig';
+
+import styles from './index.module.css';
+
+const Footer: FC = () => {
+  const { footerLinks, socialLinks } = useSiteConfig();
+
+  return (
+    <div className={styles.footer}>
+      <div className={styles.left}>
```

Comment:
nit: Classifying the element based on its current position(`left`, `right`) can be misleading. This situation is changing for mobile, and the design may also change in the future. It would be more understandable to name it based on the elements it covers rather than its position

Reply:
renamed sections to `primary` and `secondary`